src/mcts/search.cc

```diff
@@ -1236,19 +1236,23 @@ void SearchWorker::DoBackupUpdateSingleNode(
     // Convert parents to terminals except the root or those already converted.
     can_convert = can_convert && p != search_->root_node_ && !p->IsTerminal();
 
-    // A non-winning terminal move needs all other moves to have the same value.
-    if (can_convert && v != 1.0f) {
+    // A non-winning terminal move needs all other moves to be similar.
+    auto all_losing = true;
+    if (can_convert && v <= 0.0f) {
       for (const auto& edge : p->Edges()) {
-        can_convert = can_convert && edge.IsTerminal() && edge.GetQ(0.0f) == v;
+        const auto Q = edge.GetQ(0.0f);
+        can_convert = can_convert && edge.IsTerminal() && Q <= 0.0f;
```

Comment:
So technically this permits declaration of draw for mix of draw and and losing terminal edges - which is probably valid - but the old code did not allow that.  Just want to make sure that is intended.

Reply:
Nod. I marked this to fix #836 implementing your suggestion back in https://github.com/LeelaChessZero/lc0/pull/822#discussion_r276053057

E.g.,
```
position fen 8/8/8/8/8/5K2/R7/7k w - - 0 1
go nodes 50000 searchmoves a2h2

Before: a2h2 N:   50135 P:  0.17% Q:  0.01117
After:  a2h2 N:   50373 P:  0.17% Q:  0.00000 (T) 

position fen 8/8/8/8/8/5K2/R7/7k w - - 0 1 moves a2h2
go nodes 1

h1g1 N:     561 P: 45.76% Q: -1.00000 (T) 
h1h2 N:   49684 P: 54.25% Q:  0.00000 (T) 
```